clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -41,6 +41,7 @@ export const COPYBOOK_CPY_EXT = ".CPY";
 export const COPYBOOK_EXT_ARRAY = [COPYBOOK_CPY_EXT];
 export const EXTENSION_ID = "broadcommfd.cobol-language-support";
 export const TELEMETRY_DEFAULT_CONTENT = "INVALID_INSTRUMENTATION_KEY";
+export const ZOWE_EXT_MISSING_MSG = "Zowe explorer plugin with version greater than equal to 15.0.0, is mandatory for copybook download or, manually place and configure the copybook into local path"
```

Comment:
@zeibura

Reply:
*1.15.0

Reply:
Hi @zeibura ,
So, there are 3 scenarios - 

1. when the user passes a profile, which doesn't exist. 
    response - user get an error pop up saying "Can't read members of dataset: <dataset>"
2. when the user doesn't configure the profile.
    response - user gets an error pop-up - "Some copybooks could not be located. Ensure your configuration contains correct paths to copybooks, including nested copybooks. Missing copybooks: "
3. when the user passes invalid cred.
 response - `Incorrect credentials in Zowe profile ${PROFILE_NAME_PLACEHOLDER}.`

Reply:
OK, I don't think the first two error messages are particularly helpful for those scenarios. It might be good to consolidate those under a new error message in the future, with something like "Invalid Zowe profile configuration"

As for this one, just change it to "Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe"

Reply:
@zeibura Have updated the error message for 1st 2 scenarios.
In case of no profile config, we give `Please specify a Zowe Explorer profile for copybook downloading.`
And in the case of the wrong profile name, we show an error message from the zowe explorer plugin.

![image](https://user-images.githubusercontent.com/69206564/147927268-18631d64-efe1-4f36-823c-cedb93723479.png)
![image](https://user-images.githubusercontent.com/69206564/147927327-b548dab0-9f53-4899-8e6b-91330028d2e4.png)

Reply:
Great, that's lots better. Just change the end from "for copybook downloading" to "to download copybooks from the mainframe"